src/cmd-buildextend-aws

```diff
@@ -41,12 +41,13 @@ buildmeta_path = os.path.join(builddir, 'meta.json')
 with open(buildmeta_path) as f:
     buildmeta = json.load(f)
 
+arch = get_basearch()
 base_name = buildmeta['name']
 if args.name_suffix:
     ami_name_version = f'{base_name}-{args.name_suffix}-{args.build}'
 else:
     ami_name_version = f'{base_name}-{args.build}'
-aws_vmdk_name = f'{ami_name_version}-aws.vmdk'
+aws_vmdk_name = f'{ami_name_version}-aws.{arch}.vmdk'
```

Comment:
Minor/optional: could just `get_basearch()` right here since it's just a single occurrence (and similarly below).

Reply:
i think I'll leave it like it is :)